pages/cloud/lake-effect.md

```diff
@@ -10,38 +10,62 @@ Although high performance computing (HPC) is our largest and most widely used se
 
 ### Lakeeffect Quick Start
```

Comment:
A nit-picky comment but suggest using the same formatting throughout the docs for our naming of things.  Mostly you have LakeEffect, here it's Lakeeffect.  Some places we have coldfront or Coldfront but we've been trying to use ColdFront everywhere